tests/client/core/reducers/test_categories.js

```diff
@@ -10,7 +17,20 @@ describe('categories reducer', () => {
   it('defaults to an empty set of categories', () => {
     const state = categories(initialState, { type: 'unrelated' });
     assert.deepEqual(state.categories, {
-      android: {}, firefox: {},
+      android: {
+        [ADDON_TYPE_DICT]: [],
+        [ADDON_TYPE_EXTENSION]: [],
+        [ADDON_TYPE_LANG]: [],
+        [ADDON_TYPE_OPENSEARCH]: [],
+        [ADDON_TYPE_THEME]: [],
+      },
+      firefox: {
+        [ADDON_TYPE_DICT]: [],
+        [ADDON_TYPE_EXTENSION]: [],
+        [ADDON_TYPE_LANG]: [],
+        [ADDON_TYPE_OPENSEARCH]: [],
+        [ADDON_TYPE_THEME]: [],
+      },
```

Comment:
Instead of hard coding this list, I think it would be better to call `emptyCategoryList()` directly. I don't think you gain much by hard coding all the values.